lib/IRGen/IRGenSIL.cpp

```diff
@@ -3567,8 +3568,11 @@ void IRGenSILFunction::visitRefTailAddrInst(RefTailAddrInst *i) {
 }
 
 static bool isInvariantAddress(SILValue v) {
-  auto root = getUnderlyingAddressRoot(v);
-  if (auto ptrRoot = dyn_cast<PointerToAddressInst>(root)) {
+  SILValue accessedAddress = getAccessedAddress(v);
+  if (accessedAddress->getType().isAddress() && isLetAddress(accessedAddress)) {
```

Comment:
I have been noticing the Let Address code interspersed through out. Was this just something that you noticed? Shouldn't it be in a different commit?

Reply:
I just happened to rename it when I moved the utility and added the assert.